mkt/translations/tests/testapp/models.py

```diff
@@ -1,30 +1,46 @@
 from django.db import models
 
-from mkt.site.models import ModelBase
-from mkt.translations.fields import (LinkifiedField, PurifiedField, save_signal,
-                                 TranslatedField)
+from queryset_transform import TransformQuerySet
 
+from mkt.translations import transformer
+from mkt.translations.fields import (LinkifiedField, PurifiedField,
+                                     save_signal, TranslatedField)
 
-class TranslatedModel(ModelBase):
+
+class ManagerWithTranslations(models.Manager):
+    def get_query_set(self):
+        qs = TransformQuerySet(self.model)
+        if hasattr(self.model._meta, 'translated_fields'):
+            qs = qs.transform(transformer.get_trans)
+        return qs
+
+
+class TranslatedModel(models.Model):
     name = TranslatedField()
     description = TranslatedField()
     default_locale = models.CharField(max_length=10)
     no_locale = TranslatedField(require_locale=False)
 
+    objects = ManagerWithTranslations()
+
 models.signals.pre_save.connect(save_signal, sender=TranslatedModel,
                                 dispatch_uid='testapp_translatedmodel')
 
 
-class UntranslatedModel(ModelBase):
+class UntranslatedModel(models.Model):
     """Make sure nothing is broken when a model doesn't have translations."""
     number = models.IntegerField()
 
+    objects = ManagerWithTranslations()
+
 
-class FancyModel(ModelBase):
+class FancyModel(models.Model):
```

Comment:
This and above are just test apps used in `mkt/translations/tests/test_models.py`